README.md

```
@@ -1,371 +1,330 @@
+## 🏗 Architecture
 
-- 🔐 **End-to-End Encryption**: Secure encrypted data transmission
-- 🏠 **Room Management**: Create and manage chat rooms with multi-user support
-- 🔄 **Real-time Communication**: Bidirectional real-time communication based on Socket.IO
-- ⚙️ **Flexible Configuration**: Environment variable configuration for ports, CORS, etc.
-- 🛡️ **Security Controls**: Built-in CORS protection, message size limits, and other security mechanisms
-- 📊 **Health Monitoring**: Provides `/health` endpoint for service monitoring
+This monorepo uses Yarn workspaces to manage multiple packages:
 
-## Tech Stack
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)